TrainingExtensions/tensorflow/src/python/aimet_tensorflow/examples/test_models_keras.py

```diff
@@ -0,0 +1,294 @@
+# /usr/bin/env python3.5
```

Comment:
Can we please put these test models module in the test directory and not the source directory?
We have already made this mistake for existing tf and torch tests that we should correct